config/locales/en.yml

```
@@ -796,3 +796,28 @@ en:
+    mark_all_as_read:
+      success: All notifications marked as read
+  subscriptions:
+    create:
+      success: Successfully subscribed to post
+    destroy:
+      success: Successfully unsubscribed from post
+    subscribe_button:
+      subscribe: Subscribe
+      unsubscribe: Unsubscribe
+      subscribed: Subscribed
+  posts:
```

YAMLlint (1.37.1)

[error] duplication of key "posts" in mapping

(key-duplicates)